src/web_interface/static/js/system_health.js

```diff
@@ -0,0 +1,160 @@
+const BOOTSTRAP_DANGER_COLOR = "#dc3545";
+const BOOTSTRAP_PRIMARY_COLOR = "#007bff";
+
+function change_button(button_id) {
+    element = document.getElementById(button_id);
+    if (element.classList.contains("fa-caret-down")) {
+        element.classList.remove("fa-caret-down");
+        element.classList.add("fa-caret-up");
+    } else {
+        element.classList.remove("fa-caret-up");
+        element.classList.add("fa-caret-down");
+    }
+}
```

Comment:
According to the HTML template, the classList is statically initialized in a proper state. Then, we could also use toggle:

```suggestion
function change_button(button_id) {
    element = document.getElementById(button_id);
    ["fa-caret-down", "fa-caret-up"].forEach(class_name => element.classList.toggle(class_name));
}
```

Reply:
Nice that's much better :smile: